.github/workflows/release.yml

```
@@ -0,0 +1,115 @@
+        ln -s /Applications dmg_contents/Applications
+        
+        # Create the DMG
+        hdiutil create -volname "Keyboard Overlay" \
+                       -srcfolder dmg_contents \
+                       -ov -format UDZO \
+                       KeyboardOverlay.dmg
+    
+    - name: Create ZIP archive
+      run: |
+        zip -r KeyboardOverlay.zip ./KeyboardOverlay.app
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)